core-bundle/contao/widgets/TextArea.php

```diff
@@ -114,18 +114,14 @@ protected function getPost($strKey)
 	 */
 	public function generate()
 	{
-		if ($this->rte)
-		{
-			$this->strClass = trim($this->strClass . ' noresize');
-		}
-
 		return \sprintf(
-			'<textarea name="%s" id="ctrl_%s" class="tl_textarea%s" rows="%s" cols="%s"%s data-action="focus->contao--scroll-offset#store" data-contao--scroll-offset-target="autoFocus">%s</textarea>%s',
+			'<textarea name="%s" id="ctrl_%s" class="tl_textarea%s" rows="%s" cols="%s"%s%s data-action="focus->contao--scroll-offset#store" data-contao--scroll-offset-target="autoFocus">%s</textarea>%s',
 			$this->strName,
 			$this->strId,
 			$this->strClass ? ' ' . $this->strClass : '',
-			$this->intRows,
+			$this->rte ? $this->intRows : 1, // Let the controller handle the height
```

Comment:
This change makes the controller handle the height 😊 as it would default to e. g. 8 rows, thus leading to at least 240px height